src/presign/participant.rs

```diff
@@ -977,12 +975,18 @@ impl PresignKeyShareAndInfo {
         sender_r1_priv: &RoundOnePrivate,
         receiver_r1_pub_broadcast: &RoundOnePublicBroadcast,
     ) -> Result<(RoundTwoPrivate, RoundTwoPublic)> {
-        // Picking betas as elements of [+- 2^384] here is like sampling them from the
-        // distribution [1, 2^256], which is akin to 2^{ell + epsilon} where ell
-        // = epsilon = 384. Note that we need q/2^epsilon to be negligible.
-        let beta = random_plusminus_by_size(rng, ELL);
-        let beta_hat = random_plusminus_by_size(rng, ELL);
-
+        let beta = random_plusminus_by_size(rng, ELL_PRIME);
+        let beta_hat = random_plusminus_by_size(rng, ELL_PRIME);
+
+        // Note: The implementation specifies that we should encrypt the negative betas here, and
```

Comment:
Suggestion: I'd recommend that you cite the relevant sections in the paper in this discussion just for documentation's sake.

Reply:
Ooh yes. Fixed!